Sources/Views/AnimatableTableViewCell.swift

```diff
@@ -156,7 +159,9 @@ open var startPoint: GradientStartPoint = .top
   fileprivate func configureInspectableProperties() {
     configureAnimatableProperties()
     configureOpacity()
+    #if os(iOS)
```

Comment:
@JakeLin I think this can be done but I have to try it first. But then we would have to expose `TableViewCellDesignable` to the tvOS target.